src/Relation/Nullary/Negation/Core.agda

```diff
@@ -0,0 +1,91 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Core properties related to negation
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Relation.Nullary.Negation.Core where
```

Comment:
Ugh, the whole `Relation.Nullary` directory needs a serious rethink at some point... Now might not be the time though.